consensus/src/transaction_shuffler/fairness/conflict_key/entry_fun_module.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::transaction_shuffler::fairness::conflict_key::ConflictKey;
+use aptos_types::transaction::{SignedTransaction, TransactionPayload};
+use move_core_types::language_storage::ModuleId;
+
+#[derive(Eq, Hash, PartialEq)]
+pub enum EntryFunModuleKey {
+    Module(ModuleId),
+    AnyScriptOrModuleBundle,
+    AnyMultiSig,
+    Exempt,
```

Comment:
Where is the `Exempt` applied?

Reply:
in `extract_from()`, only exempting 0x1 for now. will tune soon